src/test/java/org/jitsi/meet/test/BandwidthEstimationTest.java

```diff
@@ -269,22 +298,67 @@ public void setupClass()
     }
 
     @Test
-    public void test10mbit60s1mbit60s10mbit60s()
+    public void att100mbit90s()
```

Comment:
check out [parameterized tests](https://github.com/junit-team/junit4/wiki/parameterized-tests) in junit, they're perfect for this sort of thing.